web/themes/custom/pineapple/sass/components/paragraphs/_accordion.scss

```diff
@@ -0,0 +1,48 @@
+.accordion-title {
+  background-color: $light-gray;
+  color: $primary;
+  width: 100%;
+  text-align: left;
+  padding: $unit *2;
+  transition: 0.4s;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &:hover,
+  &:focus {
+    color: $white;
+    background-color: $dark-gray;
```

Comment:
also i think you should add a border: 1px solid $dark-gray; here it's very subtile but currently when you hover on the title there's a slight indent, add the border would rectify this.